package/skelcd-control-openSUSE.spec

```diff
@@ -59,6 +59,8 @@ Requires:       yast2-firewall
 Requires:       yast2-installation >= 3.1.201
 Requires:       yast2-iscsi-client
 Requires:       yast2-kdump
+# yast2-kdump has only runtime dependency but the package is also needed in the inst-sys
+Requires:       kdump
```

Comment:
Cannot this be a problem in ppc?

Asking because the dependency in the yast2-kdump package looked like this:

```
%ifarch ppc
Recommends:     kdump
%else
Requires:       kdump
%endif
```

Reply:
Replied on IRC: "_"ppc" means PPC32 which we do not support anymore_".

Reply:
The "ppc" actually means PPC32 which we do not support for quite long time. So I simplified the dependency.